ec2uploadimg

```diff
@@ -704,10 +701,16 @@ def get_vpc_subnet_id(args, config, region, setup,  logger):
                 '--vpc-subnet-id'
             )
         except Exception:
+            # falls back to setup.create_vpc_subnet()
             msg = 'Not using a subnet-id, none given on the '
             msg += 'command line and none found in config for '
             msg += '"subnet_id_%s" value' % region
             logger.error(msg)
+
```

Comment:
The message is now an info message since we will proceed to create a subnet to use. And the message is misleading since it states "Not using a subnet-id" but that is not true, we wil create a subnet. As such we should not write a log message here and just `pass`

Also note the if condition on line 716 is now superfluous. If we cannot create a subnet we exit on line 714. Note that clean up might be required before exit.